tox.ini

```diff
@@ -39,13 +41,22 @@ commands =
     python -c 'import pathlib; print("documentation available under file://\{0\}".format(pathlib.Path(r"{toxworkdir}") / "docs_out" / "index.html"))'
 
 # Distro
-[testenv:dist]
+[testenv:dist-{win, linux}]
 skip_install = true
-# Have to use /bin/bash or the `*` will cause that argument to get quoted by the tox command line...
+platform =
+    win: win|msys
+    linux: linux|darwin
+allowlist_externals = 
+    win: powershell
+    linux: bash  
+
 commands =
     python setup.py sdist --dist-dir={distdir} bdist_wheel --dist-dir={distdir}
-    /bin/bash -c 'python -m pip install -U --force-reinstall {distdir}/nbclient*.whl'
-    /bin/bash -c 'python -m pip install -U --force-reinstall --no-deps {distdir}/nbclient*.tar.gz'
+    win: powershell Get-ChildItem -Path {distdir} -Filter nbclient*.whl | foreach \{python -m pip install -U --force-reinstall $_.FullName\}
+    win: powershell Get-ChildItem -Path {distdir} -Filter nbclient*.tar.gz | foreach \{python -m pip install -U --force-reinstall --no-deps $_.FullName\}
+    # Have to use /bin/bash or the `*` will cause that argument to get quoted by the tox command line...
+    linux: bash -c 'python -m pip install -U --force-reinstall {distdir}/nbclient*.whl'
+    linux: bash -c 'python -m pip install -U --force-reinstall --no-deps {distdir}/nbclient*.tar.gz'
```

Comment:
We don't need to use `/bin/bash` anymore?

Reply:
The bash should be in the path. Even cleaner would be to use just `sh` 
`linux: sh -c 'python -m pip install -U --force-reinstall {distdir}/nbclient*.whl'` etc.